base/applications/sndvol32/lang/bg-BG.rc

```diff
@@ -53,6 +53,8 @@ BEGIN
     CONTROL "", -1, "static", SS_ETCHEDHORZ | WS_CHILD | WS_VISIBLE, 4, 19, 82, 1
     LTEXT "Ляво/Дясно", -1, 10, 25, 80, 8
     CONTROL "", IDC_LINE_SLIDER_HORZ, "msctls_trackbar32", TBS_HORZ | WS_CHILD | WS_VISIBLE | WS_TABSTOP, 25, 35, 40, 17
+    CONTROL IDI_SPEAKER_RIGHT, -1, "static", SS_ICON | SS_REALSIZEIMAGE | WS_CHILD | WS_VISIBLE, 14, 38, 10, 10 
+    CONTROL IDI_SPEAKER_LEFT, -1, "static", SS_ICON | SS_REALSIZEIMAGE | WS_CHILD | WS_VISIBLE, 66, 38, 10, 10 
```

Comment:
Syntax is: `CONTROL "", IDI_SPEAKER_LEFT, "static", ...`
See https://docs.microsoft.com/en-us/windows/desktop/menurc/control-control for more details.
Therefore I have no idea why it works in Win7+.

Reply:
Without `SS_REALSIZEIMAGE` on Windows:

![image](https://user-images.githubusercontent.com/578406/53025053-509e2100-3471-11e9-9f42-e02eefa4ee6d.png)

On ReactOS still no icon displayed.

-----

Using `ICON IDI_ICON, -1, x, y [, width, height]` both on Windows and ReactOS:

![image](https://user-images.githubusercontent.com/578406/53025127-6a3f6880-3471-11e9-95be-58a2f1455b3d.png)

Reply:
This is because inside `AddDialogControl` the dialog resource is parsed, and it is parsing it wrong.

Reply:
I'm not sure it's clever to implement such function in the first place.

Reply:
Yeah I don't see why it would need to do this,
makes it more complex and error-prone (as shown).

Reply:
![image](https://user-images.githubusercontent.com/578406/53045827-abe70800-349f-11e9-823c-cd70cb324a29.png)

Everything works now thanks to @learn-more's patch! :tada: